workflows/schema.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional
+from pandera.polars import PolarsData
+import pandera.polars as pa
+from pandera.typing.polars import DataFrame, Series
+import polars as pl
+
+
+class RawBars(pa.DataFrameModel):
+    ticker: Optional[Series[str]] = pa.Field()
+    open: Optional[Series[float]] = pa.Field(ge=0)
+    high: Optional[Series[float]] = pa.Field(ge=0)
+    low: Optional[Series[float]] = pa.Field(ge=0)
+    close: Optional[Series[float]] = pa.Field(ge=0)
```

Comment:
The names of these have `_price` in the code which might be an issue I'm not 100% sure.